documentation/source/user_section/tutorials/gm-wm/gm-wm-metric-computation/extract-mtr-in-specific-tracts.rst

```diff
@@ -0,0 +1,56 @@
+Using the atlas to extract MTR from specific white matter tracts
+################################################################
+
+In the following examples, we will compute MTR within specific sub-regions of the white matter.
+
+Corticospinal tracts (averaged across 16 slices)
+------------------------------------------------
+
+First, we extract the MTR from the right and left corticospinal tracts.
+
+.. code::
+
+   sct_extract_metric -i mtr.nii.gz -f label/atlas -method map -l 4,5 -z 5:15 -o mtr_in_cst.csv
+
+:Input arguments:
+   - ``-i`` : Image to extract values from.
+   - ``-f`` : File or folder used to pick out specific regions from the input image. In this case, we supply the folder of the white matter atlas that was transformed to the space of the MTR data.
+   - ``-method`` : We specify ``map`` to choose the Maximum a Posteriori method, which helps to account for the partial volume effect. This method is most suitable for small regions. This method should only be used with the PAM50 white/gray matter atlas (or with any custom atlas) as long as the sum across all overlapping labels equals 1 in each voxel part of the atlas.
+   - ``-l`` : The IDs of the label (or group of labels) to compute metrics for. In this case, labels 4 and 5 correspond to the left and right lateral corticospinal tracts. You can see the full list of labels to choose from by running ``sct_extract_metric -list-labels``.
+   - ``-z`` : This argument further restricts the metric computation to specific slices. In this case, 16 slices in total (5:15) will be used for the averaging process.
+   - ``-o`` : The name of the output file.
+
+.. csv-table:: ``mtr_in_cst.csv``: MTR values in corticospinal tracts
+   :file: mtr_in_cst.csv
+   :header-rows: 1
+
+.. TODO: In the SCT Course video, "Size[vox]" used to say ~1500 because it included the voxels used in all slices. Now, it shows 67/68, which I assume is the average size across all of the slices. Is this change expected? When did we make this change?
```

Comment:
Reminder to check whether this behavior change was expected or not...

Reply:
I think this might have been due to a `-method map` bug which has since been fixed: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3216